docs/source/user/aerodyn/driver.rst

```diff
@@ -0,0 +1,497 @@
+
+An example of inputs is given below:
+
+.. code::
+
+    ----- Inflow Data ---------------------------------------------------------------
+              0   CompInflow  - Compute inflow wind velocities (switch) {0=Steady Wind; 1=InflowWind}
+    "unused"      InflowFile  - Name of the InflowWind input file [used only when CompInflow=1]
+            9.0   HWindSpeed  - Horizontal wind speed   [used only when CompInflow=0 and AnalysisType=1] (m/s)
+            140   RefHt       - Reference height for horizontal wind speed [used only when CompInflow=0]  (m)
+           0.10   PLExp       - Power law exponent   [used only when CompInflow=0 and AnalysisType=1]                        (-)
+
+
+
+**Turbine data**
+
+The user specify the number of turbines as follows:
+
+.. code:: 
+
+    ----- Turbine Data --------------------------------------------------------------
+    1   NumTurbines  - Number of turbines (should be 1 for AnalysisType=2 or AnalysisType=3)
+
+As noted in the comment, the number of turbine should be 1 for `AnalysisType=2` and `AnalysisType=3`.
+After this section, the geometry and motion is provided for each turbine. Inputs for each turbine must have the suffix `(i)` where `i` is the turbine number.
+Outputs for each turbine will be written to distinct files, with the suffix `.Ti` where `i` is the turbine number.
+
+An example of configuration with two wind turbines is shown in :numref:`fig:MultiRotor`. The figure defines the different frames and origin associated with each turbine: the turbine base frame (t), nacelle frame (n), hub frame (h), and blade frames (b). The notations and conventions follow the OpenFAST frames, except that the turbine frame does not have its origin at the tower base.
+Prescribed motions of the turbine occur at the turbine origin.
+Yawing occurs around the :math:`z_n` axis,  the blade rotates about the :math:`x_h` axis, and pitching occurs around the individual :math:`z_b` axes. The definitions of the different frames are standardized when using a basic (HAWT) input format definition, and are arbitrarily defined when using the advanced input format. More details are given in the next paragraph.
+
+.. figure:: figs/MultiRotor.png
+   :width: 80%
+   :name: fig:MultiRotor
+           
+   Definition of multiple rotors. 
+
+
+
+
+**Turbine geometry definition**
+
+Two turbine input formats are supported:
+
+- basic (`BasicHAWTFormat=True`): Basic horizontal axis wind turbine (HAWT) format. In this format, the turbine geometry is entirely determined by the number of blades (`NumBlades`), the hub radius (`HubRad`), the hub height (`HubHt`), the overhang (`Overhang`), the shaft tilt (`ShftTilt`) and the precone (`Precone`). The definition of each parameter follows the OpenFAST convention. Additionally, the user needs to provide the origin of the turbine base at `t=0` (`BaseOriginInit`).
```

Comment:
I would say OpenFAST does not have a convention; perhaps you mean "ElastoDyn convention"?